### DISCUSSION TITLE
Limitations?

### DISCUSSION THREAD

**Discussion by kivox** [kata-containers/kata-containers#3455] [Q&A] [answered]
Just wondering, are the limitations that were present in kata containers v1 still present, the main limitation I am interested in is custom networking, v1 was unable to handle custom networks and support for swarm was limited, is this limitation still present or does kata now support custom networking?

Thanks in advance!

**Comment by devimc** [comment, marked as answer]
kata 2.0 doesn't support docker  hence swarm is not supported, not sure what you mean with "custom networks" take a look to the [networking document](https://github.com/kata-containers/kata-containers/blob/main/docs/design/architecture/networking.md)
